massiv-examples/vision/app/AvgSum.hs

```diff
@@ -30,3 +30,21 @@ sum3x3Filter = makeConvolutionStencil (Sz (3 :. 3)) (1 :. 1) $ \ get ->
 {-# INLINE sum3x3Filter #-}
 
 
+
+main :: IO ()
+main = do
+  let arr = computeAs S $ arrLightIx2 Par (600 :. 800)
+      toImage ::
+           (Functor (Array r Ix2), Load r Ix2 (Pixel Y' Word8))
+        => Array r Ix2 Double
+        -> Image S Y' Word8
+      toImage = computeAs S . fmap (PixelY' . toWord8)
```

Comment:
It seems like the sample in the README (here https://github.com/lehins/massiv/#massiv-io) need to be updated too.

Reply:
Good catch!! Done